clients/ui/bff/internal/mocks/static_data_mock.go

```diff
@@ -260,3 +261,529 @@ func GenerateMockArtifact() openapi.Artifact {
 	}
 	return mockData
 }
+
+func GetCatalogModelMocks() []models.CatalogModel {
+	redHatModel1 := models.CatalogModel{
+		Name:        "granite-8b-code-instruct",
+		Description: stringToPointer("Granite-8B-Code-Instruct is a 8B parameter model fine tuned from\nGranite-8B-Code-Base on a combination of permissively licensed instruction\ndata to enhance instruction following capabilities including logical\nreasoning and problem-solving skills."),
+		Provider:    stringToPointer("IBM"),
```

Comment:
Is it okay to keep these name(IBM, Red Hat etc) in mocks, or should I replace them with some random names(may be like provider1, provider2, etc)

Reply:
Hey @ppadti I'm sorry I missed this... based on [this thread](https://redhat-internal.slack.com/archives/C07BFD5J4CB/p1756220810537579) it sounds like we will need to strip references to red hat, rhelai and ibm out of these mocks. sorry for the delay :(